examples/prometheus/rules/kube.rules

```diff
@@ -0,0 +1,68 @@
+groups:
+- name: kubernetes-rules
+  rules:
+
+    - alert: DockerLatencyHigh
+      expr: max(kubelet_docker_operations_latency_microseconds{quantile="0.9"}) / 1e+06 > 1
```

Comment:
would be nice to have this alert per node:
`max(kubelet_docker_operations_latency_microseconds{quantile="0.9"}) by (instance) / 1e+06`

(and use $instance in the description)

Reply:
Good catch. Thanks.